### PR TITLE
READMEs: Update + fix typos + reformat

### PR DESCRIPTION
**Pull request by ace-n**
(I didn't update the SLAVE instructions, just the MASTER ones.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,7 +59,8 @@ Set the project the gcloud command will use.
 Create the cluster.
 
 * `gcloud beta container clusters create jenkins`
```
Comment (waprin) [inline README.md:61]: we can take out beta now

File: README.md [hunk 2]
```diff
@@ -59,7 +59,8 @@ Set the project the gcloud command will use.
...
-Make sure the kubectl is using this new cluster.
+
+Make sure kubectl is using this new cluster.
 
 * `gcloud beta container clusters get-credentials jenkins`
```
Comment (waprin) [inline README.md:66]: take out beta here too

File: README.md [hunk 3]
```diff
@@ -69,28 +70,31 @@ This is a helper script that creates a temporary GCE instance in order to format
 
 * `./create_and_format_disk.sh`
```
Comment (waprin) [inline README.md:71]: This is no longer necessary

File: setup/master/README.md [hunk 4]
```diff
@@ -67,41 +67,39 @@ gcloud compute instances delete temp-writer
...
+   ```
+ 
+8. You should now be able to access your Jenkins webserver! Find the IP under the **EXTERNAL-IP** column by running the following command:
    ```
-gcloud compute forwarding-rules list
+kubectl get services
 ```
 
-   Alternatively if you want to maximize uptime in the event of pod deletion, you can create a [replication controller](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/replication-controller.md)of size 1
+   Alternatively if you want to maximize uptime in the event of pod deletion, you can create a [replication controller](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/replication-controller.md) of size 1
```
Comment (waprin) [inline setup/master/README.md:98]: we need to switch this from replication controller to deployment but its ok for now

**Review by waprin — CHANGES_REQUESTED** [review]
Look good, let's clean up a few of those beta commands then we'll merge. Thanks for taking a a pass at this, definitely will still need a bit more work.

**Comment by tswast** [comment]
FYI: I'm merging this so we can archive this repo in a somewhat better state.

**Pull request merged**